Userland/Libraries/LibWeb/HTML/Storage.cpp

```diff
@@ -182,7 +182,7 @@ JS::Value Storage::named_item_value(FlyString const& name) const
 {
     auto value = get_item(name);
     if (!value.has_value())
-        return JS::js_null();
+        return JS::js_undefined();
```

Comment:
Given that we're now deviating from the spec, could you add a comment with references explaining why we're doing this?